css-three-column-layout/styles.css

```diff
@@ -0,0 +1,56 @@
+body {
+  background-color: #1996d9;
+}
+
+img {
+  max-width: 55px;
+  margin-top: 10px;
+  margin-left: 5px;
+}
+
+h3 {
+  text-align: center;
+}
+
+p {
+  font-size: 16px;
+}
+
+.container {
+  margin-left: 10%;
+  margin-right: 10%;
+  margin-top: 20px;
+}
+
+.circle-background {
+  max-width: 80px;
+  padding: 10px;
+  background-color: white;
+  border-radius: 50%;
+}
+
+.row {
+  display: flex;
+}
+
+.column-full {
+  width: 100%;
+  font-family: 'Lato', sans-serif;
+  color: white;
+  text-align: center;
+}
+
+.column-one-third {
+  display: flex;
+  align-items: center;
+  flex-direction: column;
```

Comment:
If your `column` is a `flexbox`, that usually means you need a `row` inside that `column`. I would restructure this to have a `row` class element inside the `column-third` element. The only code on a column class should be a `width` or `flex-basis`